packages/react-dom/src/client/getActiveElement.js

```diff
@@ -9,7 +9,7 @@
 
 export default function getActiveElement(doc: ?Document): ?Element {
   doc = doc || (typeof document !== 'undefined' ? document : undefined);
-  if (typeof doc === 'undefined') {
+  if (doc == null) {
```

Comment:
We try to be as explicit as possible so if we want to check for `null` and `undefined` something like this would be better:

```suggestion
  if (doc === null || doc === undefined) {
```

Reply:
Done.